wiki/Client/Beatmap_editor/Compose/en.md

```diff
@@ -1,351 +1,212 @@
-| Distance Snap (`Y`) | **`Alt` (Hold):** Temporary toggle. Switch Beat Snap Divisor to Distance Snap **`Alt` + Mousewheel:** Adjust Distance Snap multiplier. | Distance between consecutive hit notes will be snapped based on their rhythmical difference. Best use while the timeline is paused. |
-| Lock Notes (`L`) | ("Select" on a Hit note) **Left-Click:** Lock this note | Lock selected note to current position and timestamp. |
+| `Grid Snap` (`T`) | **Hold** `Shift`: Temporary toggle. **Hold** `Ctrl`: Temporarily disable angle snapping. | Hit objects will be snapped to the grid by default. |
+| `Distance Snap` (`Y`) | **Hold** `Alt`: Temporary toggle. Switch Beat Snap Divisor to Distance Snap. `Alt` **+** `Mousewheel`: Adjust Distance Snap multiplier. | Distance between consecutive objects will be snapped based on their rhythmical difference. Best to use while the timeline is paused. |
+| `Lock Notes` (`L`) | **(Select a hit object)** `Left click`: Lock this object. | Lock selected object to current position and timestamp. |
+
+### Song's timeline
 
-### Bottom (Song's Timeline)
+![](img/song-timeline.jpg "Song's timeline")
 
-![Song's Timeline](img/BE_HSAT.jpg "Song's Timeline")
+On the **bottom-left**, it shows the **timestamp** in milliseconds and the **song duration** in percent. The percentage may be switched to "intro" or "outro" if there is storyboarding before or after the music.
 
-At the **bottom-left**, there is the **timestamp** in milliseconds (ms) and **song duration** in percent. The percentage may be switched to "intro" or "outro" if there is storyboarding before or after the music.
+On the **bottom-centre**, it shows the **timeline with markings** and the **compulsory music player buttons**. The `Test` button to the right of the timeline will save the beatmap and then allows for playtesting it, starting at the current timestamp.
 
-At the **bottom-centre**, there is a **timeline** with markings and the compulsory music player buttons. The `Test` button to the right of the timeline will save your beatmap then allows you to play test it, starting at the current timestamp.
+On the **bottom-right**, the **playback rate** can be adjusted by the value of a quarter.
 
 #### Colour markers
 
 | Colour | Description |
 | :-- | :-- |
-| Bright White Long | Current test section |
-| Yellow Long | Preview point. |
-| Yellow up | Start of Drain time. |
-| Green up | Inherited Points. (See [Timing Setup](/wiki/Client/Beatmap_editor/Timing)) |
-| Red up | Timing Points. (See Timing Setup) |
-| Blue Down | Bookmark |
+| White (long) | Current timestamp. |
+| Yellow (long tick) | Preview point. |
+| Yellow (up tick) | Start of the [drain time](/wiki/Beatmap/Drain_time). |
+| Green (up tick) | Inherited points. *See: [Timing tab](/wiki/Client/Beatmap_editor/Timing)* |
+| Red (up tick) | Timing points. *See: [Timing tab](/wiki/Client/Beatmap_editor/Timing)* |
+| Blue (down tick) | Bookmarks. |
```

Comment:
"start of the drain time" isn't market on the bottom timeline -- my guess is that the author mistook a combination of red and green timing points as something special (it isn't)
```suggestion
| White | Current timestamp |
| Yellow  | Preview point |
| Green | [Inherited timing point](/wiki/Client/Beatmap_editor/Timing#inherited-timing-point) |
| Red | [Uninherited timing point](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) |
| Blue | Bookmark |
```